ibis/backends/base/__init__.py

```diff
@@ -2,10 +2,14 @@
 
 import abc
 import functools
+import glob
+import itertools
 import re
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Callable, ClassVar, Iterable, Mapping
 
+import bracex
```

Comment:
is this a light dep?

Reply:
The current release of `bracex` has no dependencies https://github.com/facelessuser/bracex.